spec/factories/saved_page_factory.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :saved_page do
+    oidc_user_id { "user-id" }
```

Comment:
Instead of `oidc_user_id { "user-id" }` you can simply have `oidc_user` here so that when we do `FactoryBot.create(:saved_page)` FactoryBot will also create an associated `oidc_user`.

Reply:
done!